superdesk/wsgi.py

```diff
@@ -0,0 +1,11 @@
+"""WSGI entrypoint.
+
+Usage via Procfile:
+
+    rest: gunicorn -c pythong:superdesk.gunicorn_config superdesk.wsgi
+
+"""
+
+from superdesk.factory import get_app
+
+application = get_app()
```

Comment:
yep that would be possible, not sure about naming it honcho, it's not honcho specific in any way.
maybe just call it `superdesk` and do it there `if __name__ == '__main__':` or something:
```
rest: gunicorn -c python:superdesk.gunicorn_config superdesk
work: celery -A superdesk worker
beat: celery -A superdesk beat --pid=
```